scripts/compare_benchmarks.py

```diff
@@ -11,16 +14,20 @@
 min_iterations = 10
 min_runtime_ns = 59 * 1000 * 1000 * 1000
 
-def format_diff(diff, inverse_colors = False):
-    select_color = lambda value, color: color if abs(value) > 0.05 else 'white'
-
+def format_diff(diff):
     diff -= 1  # adapt to show change in percent
     if diff < 0.0:
-        color = 'green' if inverse_colors else 'red'
-        return colored("{0:.0%}".format(diff), select_color(diff, color))
+        return f"{diff:.0%}"
     else:
-        color = 'red' if inverse_colors else 'green'
-        return colored("+{0:.0%}".format(diff), select_color(diff, color))
+        return f"+{diff:.0%}"
+
+def color_diff(diff, inverse_colors = False):
+    select_color = lambda value, color: color if abs(value - 1) > 0.05 else 'white'
+
+    diff_str = format_diff(diff)
+    color = 'green' if (diff_str[0] == '+') != (inverse_colors) else 'red'
+
+    return colored(format_diff(diff), select_color(diff, color))
 
 def geometric_mean(values):
```

Comment:
from statistics import geometric_mean

Reply:
`New in version 3.8.` (https://docs.python.org/3/library/statistics.html#statistics.geometric_mean)

Part of Ubuntu 20.04 but not of MacOS.  Personally, I'd say no for that reason.